### PR TITLE
Make logging proper

### PR DESCRIPTION
**Pull request by frankrousseau**
To match other application logging style, I changed local logger by printit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,15 +25,16 @@
...
+    "async": "0.9.0",
+    "biscotto": "2.3.1",
+    "coffee-jshint": "0.2.2",
     "coffee-script": "1.8.0",
+    "coffeelint": "1.8.1",
+    "lodash.isequal": "2.4.1",
     "mocha": "1.21.5",
-    "biscotto": "2.3.1",
-    "async": "0.9.0",
+    "printit": "^0.1.6",
```
Comment (aenario) [inline package.json]: remove the `^`

File: src/utils/type_checking.coffee [hunk 2]
```diff
@@ -95,11 +97,11 @@ exports.castObject = castObject = (raw, schema, target = {}) ->
 
     if reportCastIgnore
         for own prop, value of raw when prop not in handled
-            console.log "Warning : cast ignored property '#{prop}'", \
-                    raw, new Error().stack
+            log.warn "Warning : cast ignored property '#{prop}'", \
```
Comment (aenario) [inline src/utils/type_checking.coffee]: This doesnt work with print it (multiple argument), does-it ?

Comment (frankrousseau) [inline src/utils/type_checking.coffee]: It can't. I opened an issue about that.

**Comment by aenario** [comment]
looks good now

**Pull request merged**